sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/JdbcIOTest.java

```diff
@@ -108,14 +112,14 @@ public static void startDatabase() throws Exception {
     dataSource.setServerName("localhost");
     dataSource.setPortNumber(port);
 
-
-    JdbcTestDataSet.createReadDataTable(dataSource);
+    JdbcTestHelper.createDataTable(dataSource, JdbcTestHelper.READ_TABLE_NAME);
```

Comment:
Is there anything gained by having `READ_TABLE_NAME` owned by `JdbcTestHelper`? It seems like the constant could live here, since you pass it to each of the helper methods (this is good IMO).

On that note, I don't think creating/deleting this table should be expensive, so I'd actually do it pseudorandomly in each test method so they cannot clobber each other. Perhaps via a `@Rule` if you were to be fancy. Otherwise it makes it so you cannot use inter-method parallelism (which I think some systems do by default)